CHANGELOG.md

```diff
@@ -13,6 +13,9 @@ Performance:
 * SecureRandom now defers loading OpenSSL until it's needed, reducing time to
   load SecureRandom.
 
+* Array dup and shift now use shared storage and their performance
```

Comment:
How about:
`Array#dup` and `Array#shift` are now constant-time operations by sharing the array storage and keeping a starting index.

Reply:
I do prefer this.